DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_jumbo_frame.py

```diff
@@ -0,0 +1,136 @@
+import pytest
+
+from dent_os_testbed.lib.bridge.bridge_link import BridgeLink
+from dent_os_testbed.constants import DEFAULT_LOGGER
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from dent_os_testbed.lib.bridge.bridge_fdb import BridgeFdb
+from dent_os_testbed.logger.Logger import AppLogger
+from dent_os_testbed.utils.test_utils.tb_utils import (
+    tb_get_all_devices,
+    tb_reload_nw_and_flush_firewall
+)
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_create_devices_and_connect,
+    tgen_utils_stop_traffic,
+    tgen_utils_stop_protocols,
+    tgen_utils_start_traffic,
+    tgen_utils_setup_streams,
+    tgen_utils_get_traffic_stats,
+    tgen_utils_get_loss,
+)
+
+pytestmark = pytest.mark.suite_functional_bridging
+
+
+@pytest.mark.asyncio
+async def test_bridging_frame_max(testbed):
+    """
+    Test Name: test_bridging_frame_max
+    Test Suite: suite_functional_bridging
+    Test Overview: This test comes to verify: max jumbo frame size.
```

Comment:
We are not verifying size. But we verifying that Jambo frames of max size are learned and forwarded.